docs/hive-catalog-with-s3-adls-gcs.md

```diff
@@ -225,13 +240,17 @@ To access S3-stored tables using Spark, you need to configure the SparkSession a
             .config("spark.sql.catalog.{hive_catalog_name}.fs.s3a.endpoint", getS3Endpoint)
             .config("spark.sql.catalog.{hive_catalog_name}.fs.s3a.impl", "org.apache.hadoop.fs.s3a.S3AFileSystem")
 
-            ## This two is for Azure Blob Storage(ADLS) only
+            // This two is for Azure Blob Storage(ADLS) only
```

Comment:
Azure Blob Storage is not ADLS, YES?

Reply:
Azure Blob Storage refers to the Azure storage concept, including ordinary Azure blob storage, ADLS, and ADSLV2.